manifests/overlays/apps/base/kustomization.yaml

```diff
@@ -12,3 +12,4 @@ resources:
   - ./pachyderm
   - ./watson-x
   - ./rhoai
+  - ./nvidia-nim
```

Comment:
Do we know if this is a partner connection? I'd imagine this is only for RHOAI and not for ODH by default 🤔 

If so, is it for Managed or Self Managed or both? 

We are reworking our manifest folder but currently it's hard to navigate. Answers to these questions will help where to put it. I'd imagine though, you want to find the anaconda stuff and place this effort next to that.

Reply:
On a side note, we may want to look at not directly including it (in the kustomization of where you want to deploy it) and manually installing it for the short term while we work out everything else.

We don't have a flag for disabling manifest files unfortunately. But we could build an overlay that includes it and you could use the DSC devFlags 🤔 Give me a moment to see if I can pen that together.

Reply:
@mpaulgreen Could you please answer Andrew's questions? Thanks.

Reply:
@xieshenzh Please reach out to PM for this. I think it will be Adam. Create this query in Pending Questions doc.

Reply:
@andrewballantyne I have rebased.
We haven't got the answers to the questions in this PR.
But it was confirmed earlier by Adam: `NIM will be a base offering, not an add-on`.